setup.py

```diff
@@ -35,6 +35,8 @@
         'pycodestyle==2.3.1',
     ],
     'test': [
+        'pytest<4.2.0',  # 4.2.0 is slow collecting tests and times out on CI.
+        'attrs<19.2.0',
```

Comment:
Please write a comment.
```suggestion
        'attrs<19.2.0',  # pytest 4.1.1 does not run with attrs==19.2.0
```